### PR TITLE
DG-270: Show spejder.dk in node teaser

### PR DESCRIPTION
**Pull request by mikkelblyme**
#### What does this PR do?
* DG-270: We show a 'Spejder.dk' tag if it is referenced in the `field_content_target_group`. There is a possibility of referencing unlimited amount of terms in this field, so I assumed in this PR that we're only interested in showing the spejder.dk. Let me know what you think.

Also, I started checking for the `term->getName()` but later changed it to check for `term->id()` instead.

#### How should this be manually tested?
Add "Spejder.dk" to the "Målgruppe" vocab and then add it to a node, that is shown in a teaser. It should only show "Spejder.dk" and whatever is referenced in `field_content_topic` (if any)

#### Any background context you want to provide?
From https://docs.google.com/document/d/1H3_o1Pbhvo3ychyrFpXsaXlKNQxlaOXZOlrjwLvSN1A/edit#

#### What are the relevant tickets?
DG-270

#### Screenshots (if appropriate)
See the "Spejder.dk" is shown here:
### node teaser
image

### also node teaser
image

### Extended teaser
image

### Events 
image

### CODE REVIEW

File: web/themes/custom/mungo/templates/content/node--extended-teaser.html.twig [hunk 1]
```diff
@@ -92,8 +92,13 @@ view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
...
         <div class="node-extended-teaser--content-container">
             <div class="node-extended-teaser--keyword">
-              {% if content.field_content_topic %}
-                {{ content.field_content_topic }}
+              {% if content.field_content_topic or has_spejderdk_target_group %}
+                <ul class="node-teaser--keyword-list">
+                  {% if has_spejderdk_target_group %}
+                    <li>{{ spejderdk_target_group_term_name }}</li>
+                  {% endif %}
+                  <li>{{ content.field_content_topic }}</li>
```
Comment (arnested) [inline web/themes/custom/mungo/templates/content/node--extended-teaser.html.twig:100]: Det bliver skævt det her hvis der er en spejderdk_target_group, men ikke en field_content_typic ... så får man en tom <li>

File: web/themes/custom/mungo/templates/content/node--teaser.html.twig [hunk 2]
```diff
@@ -95,8 +95,13 @@ view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
...
         <div class="node-teaser--keyword">
           {% if node.bundle in ['article', 'section_page', 'activity', 'redirect_page'] %}
-            {% if content.field_content_topic %}
-              {{ content.field_content_topic }}
+            {% if content.field_content_topic or has_spejderdk_target_group %}
+              <ul class="node-teaser--keyword-list">
+                {% if has_spejderdk_target_group %}
+                  <li>{{ spejderdk_target_group_term_name }}</li>
+                {% endif %}
+                <li>{{ content.field_content_topic }}</li>
```
Comment (arnested) [inline web/themes/custom/mungo/templates/content/node--teaser.html.twig:103]: Samme som den anden kommentar.

**Review by arnested — CHANGES_REQUESTED** [review]
To kommentarer (eller den samme to steder) i koden.

Desuden: når der er to tags bør vi så smide en dot eller tankestreg i mellem for at skille dem tydeligere end med spacing?

File: web/themes/custom/mungo/templates/content/node--extended-teaser.html.twig (hunk 1 above)
Comment (mikkelblyme) [reply web/themes/custom/mungo/templates/content/node--extended-teaser.html.twig:100]: Jeg har rettet til i mit seneste commit 👍 
`field_content_topic` er required på noden
image

**Review by arnested — APPROVED** [review]
LGTM :+1:

**Comment by arnested** [comment]
Nu hvor navnet er valgt til at være Track kunne det være man skulle bruge det i steder for spejder.dk i koden?

**Comment by mikkelblyme** [comment]
backstop-check approve

**Pull request merged**